spec/index.md

```diff
@@ -1088,13 +1088,13 @@ During these steps, no other entry point of the old or new canister is invoked.
 
 These steps are atomic: If `canister_pre_upgrade` or `canister_post_upgrade` trap, the upgrade has failed, and the canister is reverted to the previous state. Otherwise, the upgrade has succeeded, and the old instance is discarded.
 
-#### Canister eviction {#system-api-canister-eviction}
+#### Canister upgrade skip pre_upgrade method {#system-api-canister-upgrade-skip-pre_upgrade}
```

Comment:
Now that it's a flag, I don't think we need this as a separate section. I'd just consolidate everything into a simple note.

```
:::note
The `skip_pre_upgrade` flag can be enabled to skip the execution of the`canister_pre_upgrade` method on the old canister instance.
The main purpose of this mode is recovery from cases when the `canister_pre_upgrade` hook traps unconditionally preventing the normal upgrade path.

Skipping the pre-upgrade can lead to data loss.
Use it only as the last resort and only if the stable memory already contains the entire canister state.
:::
```

Reply:
Done, PTAL